public/css/icinga/forms.less

```diff
@@ -517,3 +517,26 @@ form.icinga-form .form-info {
 .flyover-content .control-label-group {
   text-align: left;
 }
+
+.control-group > label {
+  text-align: center;
+}
+
+.theme-mode-input {
+  display: none;
+  text-align: center;
```

Comment:
Doesn't make any sense, the input is hidden, remove this.

Reply:
✅